recipe/doc/recipe_guide.md

```diff
@@ -17,35 +17,46 @@ This guide assumes that you already know how to write and compile code that supp
 If your library is properly configured as such, you will need to do a bit of extra work to ensure that your conda package supports this as well:
 
 - You must [ignore the `run_exports`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements) of any CUDA packages that your package depends on. Otherwise, the `run_exports` would require the runtime libraries to have versions equal to or greater than the versions used to build the package.
-- You must add explicit runtime dependencies (or `run_constrained` for soft/optional dependencies) that specify the desired version range for the dependencies whose `run_exports` have been ignored. 
+- You must add explicit runtime dependencies (or `run_constrained` for soft/optional dependencies) that specify the desired version range for the dependencies whose `run_exports` have been ignored.
 
-As an example, consider that you have built a package that requires `libcublas`:
+As an example, consider that you have built a package that requires `cuda-cudart`:
```

Comment:
Switched from `libcublas` to `cuda-cudart` because I think it's a more common need.

Reply:
My 2c. I feel using `cuda-cudart` here is nerve wrecking, for two reasons,
1. When building a binary containing CUDA code, once cudart is statically linked the MVC is likely already ensured (with some caveats). Any extra treatment only makes the story more complicated.
2. When using `{{ compiler('cuda') }}` it also comes with cudart installed to the build environment. It makes it harder to learn from this example via cross-referencing a conda-build output log.

Reply:
Ah, that is fair. I had not considered that cudart is often statically linked. I'll go back to libcublas.

Reply:
Fixed in 48c3564d262ed46432e36e1acbd6c35713801b35.